docusaurus-docs/community/policies/backwards-compatibility.md

```diff
@@ -1,9 +1,280 @@
+changes to the database can be destructive to data and prevent rolling back to
+earlier versions of conda-store. To maintain backward compatibility we follow
+these principles:
+
+- New columns or tables should be added instead of removing or altering existing
+ones.  - Columns and tables should not be renamed. Aliases should be used for
+poorly named existing columns or tables.
+
+### REST API endpoints
+
+REST API endpoints are versioned on a per-endpoint basis. New endpoints will
+start at `v1`.
+
+Non-breaking changes do not require a new version of an endpoint. For REST API
+endpoints, examples on non-breaking changes are:
+
+- adding a parameter to the return value of an endpoint - making a previously
+mandatory input optional.
+
+These changes can be done without a new endpoint version.
+
+However, changes such as:
+
+- removing a parameter from the return value - altering the meaning of a value -
+making a formerly optional parameter mandatory
+
+are breaking changes and **require** a new endpoint version.
+
+When a new version of an endpoint is created, then all new features will be
+added to the new version. 
+
+Older versions of API endpoints are still considered supported and will receive
+bug fixes and security updates to their features but new features will not be
+backported to them.
+
+#### Experimental changes
+
+conda-store will expose experimental features within the `experimental`
+namespace.
+
+For example, if a new version of the `example.com/api/v1/user` endpoint is being
+tested, but not yet considered stable, it can be made available at the
+`example.com/api/experimental/user` route.  This allows conda-store contributors
+to test new changes and get community feedback without committing to supporting
+a new version of an API endpoint.  Using the `experimental` namespace is not
+mandatory. However, deploying a versioned endpoint expresses a commitment to
+support that code going forward, so it is highly recommended that developers use
+the `experimental` namespace to test new endpoints and features before marking
+them as stable.
+
+Experimental routes have no guarantees attached to them, they can be removed or
+changed at any time without warning. This allows testing features with users in
+real-world scenarios without needing to commit to support that feature as is.
+
+Once endpoints are determined to be stable and functional, they will be moved
+into the existing latest version of the API endpoint for non-breaking changes or
+a new version for breaking changes.
+
+#### Example HTTP routes
+
+##### Versioned stable route
+
+``` https://example.com/api/v1/user https://example.com/api/v2/user ```
+
+Explanation: This route has breaking changes between `v1` and `v2`.  Code
+written for the `v1` endpoint will continue to function, but all new features
+will only be available in the `v2` version of the endpoint.  This empowers users
+to upgrade when they wish to rather than forcing them to do so.
```

Comment:
```suggestion
Explanation: This route has breaking changes between `v1` and `v2`. Code
written for the `v1` endpoint will continue to function, but all new features
will only be available in the `v2` version of the endpoint. This empowers users
to upgrade when they wish to rather than forcing them to do so.
```